inscriptions/ISic000547.xml

```diff
@@ -199,88 +204,23 @@
-                        <ex>elio</ex>
-                    </expan>
-                    Rustico
-                    <expan>
-                        <abbr>mil</abbr>
-                        <ex>itavit</ex>
-                    </expan>
-                    <lb n="3"/>
-                    <expan>
-                        <abbr>ann</abbr>
-                        <ex>os</ex>
-                    </expan>
-                    VIII
-                    <expan>
-                        <abbr>tr</abbr>
-                        <ex>iere</ex>
-                    </expan>
-                    <lb n="4"/>
-                    <g ref="#centuria">centuria</g>
-                    Zeno<lb n="5" break="no"/>nis
-                    <expan>
-                        <abbr>Mar</abbr>
-                        <ex>cus</ex>
-                    </expan>
-                    Germanici<lb n="6" break="no"/>anus
-                    <expan>
-                        <abbr>mil</abbr>
-                        <ex>itavit</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ann</abbr>
-                        <ex>os</ex>
-                    </expan>
-                    X
-                    <lb n="7"/>
-                    <expan>
-                        <abbr>Valer</abbr>
-                        <ex>ius</ex>
-                    </expan>
-                    Victorinus
-                    <lb n="8"/>
-                    <expan>
-                        <abbr>her</abbr>
-                        <ex>es</ex>
-                    </expan>
-                    <expan>
-                        <abbr>b</abbr>
-                        <ex>ene</ex>
-                    </expan>
-                    <expan>
-                        <abbr>m</abbr>
-                        <ex>erenti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>f</abbr>
-                        <ex>ecit</ex>
-                    </expan>
-                    .
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><expan><abbr><supplied reason="lost">A</supplied>ur</abbr><ex>elio</ex> <g ref="#interpunct">·</g> </expan>Rustico<expan> <g ref="#interpunct">·</g> <abbr>mil</abbr><ex>itavit</ex></expan>
+                    <lb n="3"/><expan><abbr>ann</abbr><ex>os</ex></expan> <g ref="#interpunct">·</g> VIII <expan><abbr>tr</abbr><ex>iere</ex></expan> <g type="centuria">centuria</g><!--glyph looks like a backwards 'Y'?--> Zeno
+                    <lb n="4" break="no"/>nis <g ref="#interpunct">·</g> <expan><abbr>Mar</abbr><ex>cus</ex> </expan> <g ref="#interpunct">·</g> Germanici
+                    <lb n="5" break="no"/>anus <g ref="#interpunct">·</g> <expan><abbr>mil</abbr><ex>itavit</ex> <g ref="#interpunct">·</g> </expan><expan><abbr>ann</abbr><ex>os</ex></expan> <g ref="#interpunct">·</g> X
```

Comment:
closing expan element for militavit should be immediately after the closing ex element, not after the interpunct.
the numeral X should have a `<num value="10">X</num>` coding